src/app/core/admin/admin-entity.service.spec.ts

```diff
@@ -1,12 +1,21 @@
 import { TestBed } from "@angular/core/testing";
 
 import { AdminEntityService } from "./admin-entity.service";
+import { EntityMapperService } from "../entity/entity-mapper/entity-mapper.service";
+import { mockEntityMapper } from "../entity/entity-mapper/mock-entity-mapper-service";
 
 describe("AdminEntityService", () => {
   let service: AdminEntityService;
 
   beforeEach(() => {
-    TestBed.configureTestingModule({});
+    TestBed.configureTestingModule({
```

Comment:
can you move the test for saving the schema and view config from admin-entity.component.spec to here?